src/cgr_gwas_qc/parsers/plink.py

```diff
@@ -73,59 +74,69 @@ def read_hwe(filename: PathLike) -> pd.DataFrame:
     )
 
 
-def read_genome(filename: PathLike) -> pd.DataFrame:
+def read_genome(filename: PathLike, required_cols=None, chunksize=100000) -> pd.DataFrame:
     """Parse PLINK's genome file format.
 
     Each row of the genome file is a pairwise combinations of
-    samples/subjects. I am unsure of how plink assigns IID order. Here I sort
-    IDs alphanumerically which will make searching for pairs easier because
-    you can assume the order.
-
-    >>> ID1, ID2 = sort([IID1, IID2])
+    samples/subjects.
 
     Returns:
         pd.DataFrame:
-            A (n x 5) table with the following columns
+            A (n x 17) table with the following columns
 
             .. csv-table::
                 :header: name, dtype, description
 
-                ID1, object, First Sample or Subject ID (alphanumerically)
-                ID2, object, Second Sample or Subject ID (alphanumerically)
```

Comment:
These should be `IID1` and `IID2`. That are not changed to `ID1` and `ID2` here – that comes in the concordance_table.py module.

Reply:
Right, revised the doctstring in 6e4fc09f15b40a3ea23c4bc5b0ba426e6da36a13. Thanks.